test/fail/ok/M0129.tc.ok

```diff
@@ -1 +1 @@
-M0129.mo:2.15-2.21: type error [M0129], unexpected system method named foobar, expected heartbeat or timer or preupgrade or postupgrade or inspect
+M0129.mo:2.15-2.21: type error [M0129], unexpected system method named foobar, expected heartbeat or timer or preupgrade or postupgrade or onLowMemory or inspect
```

Comment:
`onLowMemory` is kind-of clumsy, what about `tightmemory`? Would be more in line with the other names...

Reply:
Yes, I see. In IC it is called `canister_on_low_wasm_memory`... Just thought to use a similar name for resemblance, but I am not fully happy either.
Maybe @crusso has some suggestion on how to name this.

Reply:
I think I prefer `onLowMemory` to `tightmemory`. I guess just `memory` if we want to keep it short and sweet like the others (https://internetcomputer.org/docs/current/motoko/main/reference/language-manual#system-fields), but onLowMemory is more meaningfull.

Reply:
What about `lowmemory`?

Reply:
Will chane to `lowmemory`

Reply:
Great. Please update the PR description too. (Also w.r.t. the delay problem.)